### DISCUSSION TITLE
Usages of `bobheadxi/deployments`

### DISCUSSION THREAD

**Discussion by bobheadxi** [bobheadxi/deployments#84] [Show and tell]
Sourcegraph search notebook showing usage by version: https://sourcegraph.com/notebooks/Tm90ZWJvb2s6MjQ3

A single Sourcegraph query showing usages of various version of this action: [`context:global uses: bobheadxi/deployments@.* file:^\.github/workflows count:all patternType:regexp`](https://sourcegraph.com/search?q=context:global+uses:+bobheadxi/deployments%40.*+file:%5E%5C.github/workflows+count:all&patternType=regexp)

I've also compiled a brief list of what seems to be popular repositories that use this action [in the README](https://github.com/bobheadxi/deployments/blob/main/README.md). Please feel free to share interesting use cases of this action if you have any! 😁

**Comment by SeanKilleen** [comment]
We use this to help us add Netlify previews to the [NUnit docs](https://github.com/nunit/docs).

Netlify previews are great, but they don't currently support our documentation tool (docfx). This tool allowed me to create Netlify previews with custom GitHub environments.

Thank you!
